client/src/components/companies/CompaniesListView.js

```diff
@@ -1,56 +1,9 @@
 import React from "react";
```

Comment:
This is also removing new code for the table list view component.

Reply:
I pulled main again into this branch. I think since this PR was before the last change, that wasn't a part of my bracnch. Hopefully, pulling main in fixes things.